tailnet/configmaps_internal_test.go

```diff
@@ -630,18 +621,10 @@ func TestConfigMaps_updatePeers_lost(t *testing.T) {
 		// OK!
 	}
 
-	// Before we update the clock again, we need to be sure the timeout has
-	// completed running. To do that, we check the new lastHandshake has been set
-	require.Eventually(t, func() bool {
-		uut.L.Lock()
-		defer uut.L.Unlock()
-		return uut.peers[p1ID].lastHandshake == lh
-	}, testutil.WaitShort, testutil.IntervalFast)
```

Comment:
Why do we no longer need this check?

Reply:
I'm glad you asked! One of the benefits of this library compared with `benbjohnson/clock` is that we can use `Advance().MustWait()` to wait for all the `AfterFunc()` functions to run.  So, we don't need to use `require.Eventually()` and peer into the internal state in order to ensure the timeout has been processed.

Reply:
Nice!